src/links.ts

```diff
@@ -45,6 +56,21 @@ export function getLinks(source: string, baseUrl: string): string[] {
 
 function parseAttr(name: string, value: string): string[] {
   switch (name) {
+    case 'style':
+      return value
```

Comment:
There's a risk here of an empty `<style>` tag, no?

Reply:
Agreed.